pages/get-involved.tsx

```diff
@@ -3,20 +3,96 @@ import {
   Button,
   Container,
   Grid,
+  List,
   Typography,
 } from '@mui/material';
 import React from 'react';
 
 import IndividualPageHead from '../components/helper/IndividualPageHead.tsx';
 import ShareButtons from '../components/helper/ShareButtons.tsx';
 import AccordionBuilder from '../components/layout/AccordionBuilder.tsx';
+import BenefitListItem from '../components/layout/BenefitListItem.tsx';
 import FactCard from '../components/layout/FactCard.tsx';
 import HeroBanner from '../components/layout/HeroBanner.tsx';
 import ImageContainer from '../components/layout/ImageContainer.tsx';
 import PaperSection from '../components/layout/PaperSection.tsx';
 import SectionContainer from '../components/layout/SectionContainer.tsx';
 import content from '../content/get-involved.ts';
 
+function BenefitsOfJoiningUs() {
+  return (
+    <SectionContainer
+      id="benefits-of-joining-us"
+      maxWidth="md"
+    >
+      <Typography variant="h2">
+        {content.benefits.header}
+      </Typography>
+      <PaperSection sx={{
+        bgcolor: 'primary.dark', color: 'primary.contrastText',
+      }}
+      >
+        <Box sx={{
+          pl: 3, py: 4,
+        }}
+        >
+          {content.benefits.body.map((text) => (
+            <Typography
+              key={`benefitsText-${text}`}
+              variant="body1"
+              sx={{ mb: 4, mr: 2 }}
+            >
+              {text}
+            </Typography>
+          ))}
+          <Grid container alignItems="flex-start">
+            <Grid item xs={5}>
+              <Typography variant="body1" paragraph>
+                {content.benefits.listHeader}
+              </Typography>
+            </Grid>
+            <Grid item xs={7} sx={{ display: { xs: 'inherit', sm: 'none' } }}>
+              <ImageContainer
+                src={content.benefits.mobileImgsrc}
+                width={95}
+                height={95}
+                alt=""
```

Comment:
Would be good to add `alt` for accessibility purpuses.

Reply:
in my opinion these images fall under the "decoration" category of the WCAG2 since they dont add any information to the page, in which case it's best practice for the alt text to be null so it will be ignored by assistive technology https://www.w3.org/WAI/WCAG22/quickref/?showtechniques=111#text-alternatives

Reply:
Yeah make sense, I read the link that you gave and did some research about this and i agree that this is more about a decorative thing.

Thx for letting me learn about this!! :tada: